src/proto/beats.proto

```diff
@@ -10,22 +11,69 @@ enum Source {
   USER = 2;
 }
 
-message Beat {
+message BpmMarker {
+  optional int32 beat_index = 1;
+  optional double bpm = 2;
+  optional Source source = 3;
+}
+
+message TimeSignature {
+  optional int32 beats_per_bar = 1 [ default = 4];
+  optional int32 note_value = 2 [ default = 4 ];
+}
+
+message TimeSignatureMarker {
+  optional int32 downbeat_index = 1;
+  optional TimeSignature time_signature = 2;
+}
+/*
+message PhraseMarker {
+  optional int32 downbeat_index = 1;
+}
+
+enum SectionType {
+  OTHER = 0;
+  INTRO = 1;
+  OUTRO = 2;
+}
+
+message Color {
+  optional int32 red = 1;
+  optional int32 green = 2;
+  optional int32 blue = 3;
```

Comment:
Shouldn't we just use a single int32 here and ignore the most significant byte? We only use 3-byte RGB anyway.

Reply:
I suppose we could, but then we would need to do some bitwise math to convert between the protobuf value and the data we need. I'd prefer to make the protobuf format more straightforward.

Reply:
Can't we just pass it to mixxx::RgbColor? I think so.

I fear that using separate int32 values opens more room for strange bugs. What happens if a RGB channel value is greater than 0xFF? What happens if a value is negative?

Reply:
This is commented out from the protobuf, so we don't need to determine this now.